packages/govern-console/src/pages/ViewDao.tsx

```diff
@@ -86,11 +87,16 @@ export default function DaoView() {
   const { chainId } = useChainId()
   const { daoAddress }: any = useParams()
   const { path } = useRouteMatch()
-  const { data, loading, error } = useQuery(DAO_QUERY, {
-    variables: {
-      name: daoAddress,
-    },
-    client: chainId === 4 ? rinkebyClient : mainnetClient,
+  const { data, isLoading: loading, error } = useQuery('DAO_DATA', async () => {
+    const res = await request(
+      chainId === 4 ? RINKEBY_SUBGRAPH_URL : MAINNET_SUBGRAPH_URL,
+      DAO_QUERY,
+      {
+        name: daoAddress,
+      },
+    )
+
+    return res
   })
```

Comment:
I think we could do this directly since `request()` returns a promise?

```suggestion
  const { data, isLoading: loading, error } = useQuery('DAO_DATA', () => request(
    chainId === 4 ? RINKEBY_SUBGRAPH_URL : MAINNET_SUBGRAPH_URL,
    DAO_QUERY,
    { name: daoAddress },
  ))
```

Reply:
Yes!